patches/extensions-common-permissions-permissions_data.cc.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/extensions/common/permissions/permissions_data.cc b/extensions/common/permissions/permissions_data.cc
+index b09be07c0db06ec79044ac2468488ad70fce5118..69ed8bdf0743dcdc8672167f246853546aa3073f 100644
+--- a/extensions/common/permissions/permissions_data.cc
++++ b/extensions/common/permissions/permissions_data.cc
+@@ -521,6 +521,11 @@ PermissionsData::PageAccess PermissionsData::CanRunOnPage(
+   if (IsRestrictedUrl(document_url, error))
+     return PageAccess::kDenied;
+ 
++  if (IsBraveRestrictedUrl(document_url, extension_id_, location_, error)) {
```

Comment:
please use a define here to make this a one line patch - `BRAVE_CAN_RUN_ON_PAGE`